.changeset/eight-dolphins-accept.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-kubernetes': patch
+'@backstage/plugin-kubernetes-backend': minor
+'@backstage/plugin-kubernetes-common': minor
+---
+
+Add new backend endpoints, service/:serviceId endpoint is deprecated, backend now requires discovery api config
```

Comment:
Is this a change that require file updates for a user upgrading the backend? In that case I think we should add those here.

For example

**Breaking:** The kubernetes backend now requires a discovery api config.
Make sure that `src/../kubernetes.ts` is updated with diff.

Reply:
Will add

Reply:
Done